slave-prod-centos7/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM docker.io/fhwendy/jenkins-slave-base-centos7:latest
```

Comment:
I don't know if there's anything here in this Dockerfile, or in the krb5.conf in this PR, that shouldn't be public or shouldn't be pushed to docker hub.

Reply:
I asked the RCM guy, it should be fine.